gradle/documentation/render-javadoc.gradle

```diff
@@ -268,6 +268,13 @@ class RenderJavadocTask extends DefaultTask {
 
     // create the directory, so relative link calculation knows that it's a directory:
     outputDir.mkdirs();
+    outputDir.listFiles().each {childFile ->
```

Comment:
what about `project.delete`?

Reply:
No idea why it is this way. Have you checked the Lucene version, it may be more up-to-date.

Reply:
I agree. This is more Gradle like. You should also check the Lucene version. Maybe it is more up to date in that regard. The version number is an input property there.

Reply:
I looked through the lucene implementation but I didn't really see this addressed either. Maybe I missed it though.

Will change to use `project.delete` though.